app/os/main.cpp

```diff
@@ -529,6 +529,15 @@ namespace watchdog {
 
 }	// namespace watchdog
 
+namespace deep_sleep {
+	auto components = std::to_array<interface::DeepSleepEnabled *>({
+		&motors::left::motor,
+		&motors::left::motor,
```

Comment:
```suggestion
		&motors::right::motor,
```

Ironiquement, comme ça n'a pas été UT on n'a pas vu la confusion